### ISSUE TITLE
Objects of classes missing

### ISSUE THREAD

**Issue by lscheffler** [fdbozzo/foxbin2prg#9]
The latest release does not store objects of classes om my comp. The vc2 file just holds the class and the properties and methods of it, but not the objects and there methods.

If switched to this version of FoxBin2PRG after a long time of just using.

Edit:

This is the _master_ branch. If I should use a more recent one, blease notify me.

**Comment by lscheffler** [comment]
I've tested around and are now on tag v1.19.48  (c7323b841efbd1752e26bef987f0564210428ed2)
Here it works as expected. Is there any syntax change? I'm puzzled. All docu I see is the old one on codeplex, but this is not of any help

**Comment by fdbozzo** [comment]
Hi Lutz!

There are no syntax changes, only bug fixes on that version.

BTW, you could update to the latest **[v1.19.49.1](https://github.com/fdbozzo/foxbin2prg/tree/v1.19.49.1)**, that have some bug fixes and enhancements (it's strange that you use v1.19.48, being that one bug fix for v1.19.49 was reported by you)

**v1.19.49**
- Replace Scripting.FileSystemObject MoveFile with Win32API equivalent
- Added support for using temporal CFG file for low-level API operations
- Bug Fix: Single-class export with "classlib.vcx::classname" syntax did not export (Lutz Scheffler)
- Added single-class import support ("library.vcx::classname::import")
- Added new single-class export syntax ("library.vcx::classname::export")
- Both single-class syntaxes doesn't require a CFG configuration
- Bug Fix: Partially corrupted frx/lbx info generated when using "&&" string inside field expressions (Alejandro A Sosa)
- sys(2023) secure temp: When sys(2023) points to "program files", then getenv("temp") is used
- Add more options to config.fpw to minimize interference from existent configs
- Feature: Save the contents of User field (Doug Hennig). Note: DevInfo is already supported

**v1.19.49.1**
BugFix: DEVINFO field is only used when doing PJX conversions. On other files throw errors

Regards!

**Comment by lscheffler** [comment]
Hi Fernando, 

I use [bin2text](https://github.com/lscheffler/bin2text) as interface on the IDE.  The version found on github.
foxbin2prg.cfg:
```
DBC_Conversion_Support: 0		&& 0=No support, 1=Generate TXT only (Diff), 2=Generate TXT and BIN (Merge)
DBF_Conversion_Support: 0		&& 0=No support, 1=Generate TXT only (Diff), 2=Generate TXT and BIN (Merge), 4=Generate TXT with DATA for DIFF (No Bidirectional!)
UseClassPerFile: 1				&& 0=One library tx2 file, 1=Multiple file.class.tx2 files / 0=Una archivo librería tx2, 1=Múltiples archivo.clase.tx2
RedirectClassPerFileToMain: 1	&& 0=Don't redirect to file.tx2, 1=Redirect to file.tx2 when selecting file.class.tx2 / 0=No redireccionar a archivo.tx2, 1=Redireccionar a archivo.tx2 cuando se seleccione archivo.clase.tx2

*************
New switch
*************
*RedirectClassType:0   && NEW: This indicates that only the selected vc2 class (classlib.*classX*.vc2) must be imported
Debug:1

```
After I've pulled last week, FoxBin2Text was generating the single files, but removed any object inside the classes like: (snippet from gitk)
```
DEFINE CLASS jvp_frmdefault AS jvp_cform OF "jvp_classes.vcx" 		&& PROGRAMM Grundeinstellungen
  	*< CLASSDATA: Baseclass="form" Timestamp="" Scale="Pixels" Uniqueid="" ProjectClassIcon="..\graphics\class_bmp\jvp.bmp" ClassIcon="..\graphics\class_bmp\jvp.bmp" />
 
-	*-- OBJECTDATA items order determines ZOrder / El orden de los items OBJECTDATA determina el ZOrder 
-	*< OBJECTDATA: ObjPath="txtErrorFax_Target" UniqueID="" Timestamp="" />
-	*< OBJECTDATA: ObjPath="txtErrorMail_Target" UniqueID="" Timestamp="" />
-	*< OBJECTDATA: ObjPath="lblErrorFax_Target" UniqueID="" Timestamp="" />
-	*< OBJECTDATA: ObjPath="lblErrorMail_Target" UniqueID="" Timestamp="" />
-	*< OBJECTDATA: ObjPath="txtUser_Path" UniqueID="" Timestamp="" />

```

This was no problem until I was regenerating - mind the problem. For some luck I have an independent backup running to find most of the stuff.
What I've also noticed that I was unable to regenerate single classes like _library.class.vc2_, the methods returned success but there was no class at all.

**Comment by lscheffler** [comment]
I've send you a mail with logging information

**Comment by fdbozzo** [comment]
Hi Lutz:

I can confirm this. Looking for a fix, please use v1.19.47 until then.

Thanks!

**Comment by fdbozzo** [comment]
Hi Lutz:

I think it's fixed in v1.19.49.2. At least in my tests it's working ok now and generate all the objects in the tx2 files with ClassPerFile.
Please, confirm it when you can.

Fix:
https://github.com/fdbozzo/foxbin2prg/tree/v1.19.49.2

**Comment by lscheffler** [comment]
Hi Fernando,
the tx2 is generated now again.
